spec/api/delete_fourth_operator_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rspec'
+require_relative '../../lib/rest_http'
+require_relative '../support/data'
+
+describe 'delete operator' do
+  @api_functionality = RestHttp.new
```

Comment:
You can put initialize code to `before(:all)` block. It's better to keep it there to avoid unpredictable behavior.